decidim-core/app/helpers/decidim/application_helper.rb

```diff
@@ -3,6 +3,9 @@
 module Decidim
   # Main module to add application-wide helpers.
   module ApplicationHelper
+    # Include decidim-comments helpers so they can be used anywhere
+    include Decidim::Comments::CommentsHelper
```

Comment:
I don't like coupling `Core` to comments as I can't find a scenario where it makes sense. Please put this in `pages`.